internal/api/web/views/pages/quiz_pages/quiz_main_question.templ

```diff
@@ -2,28 +2,47 @@ package quiz_pages
 
 import (
 	"github.com/Molnes/Nyhetsjeger/internal/api/web/views/components"
+	"github.com/Molnes/Nyhetsjeger/internal/api/web/views/components/quiz_components"
+	"github.com/Molnes/Nyhetsjeger/internal/data/questions"
+	"github.com/Molnes/Nyhetsjeger/internal/api/web/views/components/icons"
 )
 
-templ QuizPage() {
+//Starts a countdown from parameter to 0.
+//Runs as a javascript function.
+script startCountdown(countdownTime int) {
+	var countDownTime = 10;
+	var x = setInterval(function() {
+		countDownTime--;
+		document.getElementById("countdown").innerHTML = countDownTime;
+		if (countDownTime <= 0) {
+			clearInterval(x);
+			document.getElementById("countdown").innerHTML = "FIN";
+		}
+	}, 1000);
+
+}
+
+//Renders the basic question layout. It includes a timer, score, question and answers.
+templ QuizPage(quiz questions.Question, title string) {
```

Comment:
quiz page? filename quiz main question? i am confused. clearer and consistant naming please

Reply:
done, also consistent not consistant